### PR TITLE
Add extra changes to make possibility add eip4844 type as a plugin [DRAFT]

### LINKED ISSUE #6986: Add extra changes to make possibility add eip4844 type as a plugin

### PR DESCRIPTION
**Pull request by avkos**
## Description

List of extra things
- add custom crypto (config)
- add transactionSchema (config)
- add Chain method
- add EIP method
- add Hardfork method

Please include a summary of the changes and be sure to follow our [Contribution Guidelines](https://github.com/web3/web3.js/blob/4.x/.github/CONTRIBUTING.md).

Fixes #6986 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran `npm run lint` with success and extended the tests and types if necessary.
- [ ] I ran `npm run test:unit` with success.
- [ ] I ran `npm run test:coverage` and my test cases cover all the lines and branches of the added code.
- [ ] I ran `npm run build` and tested `dist/web3.min.js` in a browser.
- [ ] I have tested my code on the live network.
- [ ] I have checked the Deploy Preview and it looks correct.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [ ] I have linked Issue(s) with this PR in "Linked Issues" menu.

### CODE REVIEW

File: packages/web3-eth-accounts/src/common/chains/sepolia.ts [hunk 1]
```diff
@@ -1,100 +0,0 @@
-export default {
-	"name": "sepolia",
```
Comment (jdevcs) [inline packages/web3-eth-accounts/src/common/chains/sepolia.ts:2]: where these configs are moved? this will be breaking change?

**Review by jdevcs — COMMENTED** [review]
lgtm , except question about [packages/web3-eth-accounts/src/common/chains/sepolia.ts](https://github.com/web3/web3.js/pull/6987/files#diff-2536f588e316795313e6f992eb9f614cdf4e3d7233c8747548ee75a0394a7066) and same for mainnet

**Comment by jdevcs** [comment]
also see list of tests failing, plus need changelog updates

**Comment by jdevcs** [comment]
● accounts › Signing and Recovery of Transaction › sign transaction
web3-eth-accounts:     Hardfork with name merge not supported
web3-eth-accounts: 
web3-eth-accounts:       306 | 		}
web3-eth-accounts:       307 | 		if (!existing) {
web3-eth-accounts:     > 308 | 			throw new Error(`Hardfork with name ${hardfork} not supported`);
web3-eth-accounts:           | 			      ^
web3-eth-accounts:       309 | 		}
web3-eth-accounts:       310 | 	}

File: packages/web3-eth-accounts/src/common/chains/sepolia.ts (hunk 1 above)
Comment (avkos) [reply packages/web3-eth-accounts/src/common/chains/sepolia.ts:2]: here is a path of all configs
packages/web3-eth-accounts/src/common/chains/index.ts

**Pull request closed without merging**